demo.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>
+    Bluegenes Pathway Visualizer
+  </title>
+
+  <!-- You can assume imjs is on the window.  -->
+  <script src="node_modules/imjs/dist/im.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+
+  
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+  
+
+  <!-- These are styles that will be inherited from bluegenes -->
+  <link rel="stylesheet" href="css/site.css">
+
+
+  <!-- These are the styles YOU implement in src/style.less -->
+  <link rel="stylesheet" href="dist/style.css">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.2.0/socket.io.js"></script>
+	<script>
+		var socket = io("http://localhost:3457");
+    socket.on('reload', function(){
+			document.location.reload();
+    });
+	</script>
+</head>
+
+<body>
+  <div class="bluegenesPathwayVisualizer" id='yourDiv' />
+  <script>
+    window.onload = function() {
+	    //The element to attach the viewer to. Must be an existing DOM element
+    	var elem = document.getElementById('yourDiv'),
+    		//InterMine service, including URL and token.
+    		imURL = {
+    			root: 'https://www.humanmine.org/humanmine',
+    			//could include token here too if we had one
+    		},
+    		//this is an example of data that could be passed to this tool be BlueGenes
+    		//in reality (outside the demo) this would be dynamic and not hard-coded
+    		// to an ID.
+    		dataToInitialiseToolWith = {
+          Gene: {
+            class: 'Gene',
+            format: 'id',
+            // right now this id corresponds to Human GATA1
+            // this is a bit fragile since IDs change with every build
+            // but we don't want to make the code more complicated than
+            // needed for the demo.
+            value: [5468, 128, 3214]
```

Comment:
To match the updated query in a different comment, you should change the value to be database IDs. Here's one you can use:

`[1004239, 1106035, 1198868, 1065607, 1209723, 1082799, 1113059, 1095311, 1036733, 1191179, 1149697, 1057691, 1230795, 1115129, 1097269, 1021179, 1183480, 1104926, 1228554, 1086740]`

As a tip, you can't get database IDs from the Intermine or Bluegenes interface, (other than copying the end of a URL to a report page) but you can if you use an API client. I usually do `pip install --user intermine`, then open a python shell with `python` and paste the *Generate Python code* from an im-table, making sure to use `query.add_view("id")` and `print(row["id"])` instead. Note that this will only work with public lists unless you authenticate.